### PR TITLE
Nintendo music official translations

### PR DESCRIPTION
**Pull request by PhantomRuby**
updated track titles from Nintendo Music:
- Pokemon Scarlet & Pokemon Violet
- Tomodachi Collection
- Wii Shop Channel
- The Legend of Zelda: Ocarina of Time

shoutouts to the other 74 commits that I explicitely discarded but ok

### CODE REVIEW

File: album/pokemon-scarlet-violet-super-music-collection.yaml [hunk 1]
```diff
@@ -1370,63 +1372,63 @@ Artists:
...
 Artists:
 - Hiromitsu Maeba
 URLs:
 - https://www.youtube.com/watch?v=445V6xQDwwc
 Referenced Tracks:
-- Ogre Oustin'!
+- Ogre Oustin'
 ---
-Track: The Mysterious Child Who Dropped a Mask
+Track: The Mysterious Child Who Dropped Their Mask
```
Comment (leo60228) [inline album/pokemon-scarlet-violet-super-music-collection.yaml:1407]: ogerpon she/they canon (real) (not fake)

File: album/pokemon-scarlet-violet-super-music-collection.yaml [hunk 2]
```diff
@@ -1894,7 +1896,7 @@ URLs:
 Referenced Tracks:
 - Mochi Mayhem
 ---
-Track: Title Screen 2
+Track: Title Screen
```
Comment (leo60228) [inline album/pokemon-scarlet-violet-super-music-collection.yaml:1888]: oh my god they actually gave two songs on the same album the exact same name. nintendo why are you like this

File: album/references-beyond-homestuck.yaml [hunk 3]
```diff
@@ -6408,6 +6408,14 @@ Duration: '2:42'
 URLs:
 - https://www.youtube.com/watch?v=T6CK7ctyDz4
 ---
+Track: Main Theme (Wii Shop Channel)
+# No official release
```
Comment (leo60228) [inline album/references-beyond-homestuck.yaml]: ::::p

**Review by leo60228 — APPROVED** [review]

File: album/pokemon-scarlet-violet-super-music-collection.yaml (hunk 2 above)
Comment (PhantomRuby) [reply album/pokemon-scarlet-violet-super-music-collection.yaml:1888]: nintendo

File: album/references-beyond-homestuck.yaml (hunk 3 above)
Comment (PhantomRuby) [reply album/references-beyond-homestuck.yaml]: Honestly don't even know what that's doing there, there have been official releases, just not localized

**Comment by PhantomRuby** [comment]
Don't merge yet, have a small thing to update today

**Comment by PhantomRuby** [comment]
updated Batʇlə! (■■■) to match its exact name, ǝ -> ə (they are different)

feel free to merge

**Pull request merged**